editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java

```diff
@@ -241,9 +243,20 @@ private void installDefaultFacets(IProject project, MavenProject mavenProject, I
 	        installCamelFacet(fproj, fpwc, camelVersion, monitor);
 	        fpwc.commitChanges(monitor);
 	        configureNature(project, mavenProject, monitor); 
+	        updateMavenProject(project, monitor);
 	    }
 	}
 	
+	 private void updateMavenProject(IProject project, IProgressMonitor monitor) throws CoreException {
+        // MANIFEST.MF is probably not built yet
+        if (project != null) {
+            // update the maven project so we start in a deployable state
+            // with a valid MANIFEST.MF built as part of the build process.
+            Job updateJob = new UpdateMavenProjectJob(new IProject[] {project });
```

Comment:
It reveals pure implementations of various m2e configurators and/or project facet configurator but we learned it by experience on the SY fix. It was the only solution that we found